File: API/SearchByTag/SearchByTag/Core/Model/Label.cs
using System;

namespace SearchByTag.Core.Model
{
    public class Label
    {
        public long Id { get; set; }
        public String Text { get; set; }
        public String Category { get; set; }
        public long IdMain { get; set; }
        public virtual Main Main { get; set; }
    }
}

File: Angular/SearchByTag/src/app/main/label/label.ts
export class Label {
    private id: number;
    get Id(){
        return this.id;
    }
    set Id(value: number){
        this.id = value;
    }
    
    private text: string;
    get Text(){
        return this.text;
    }
    set Text(value: string){
        this.text = value;
    }

    private category: string;
    get Category(){
        return this.category;
    }
    set Category(value: string){
        this.category = value;
    }

    private idMain: number;
    get IdMain(){
        return this.idMain;
    }
    set IdMain(value: number){
        this.idMain = value;
    }

}